x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/edit_connector_flyout/index.test.tsx

```diff
@@ -444,6 +444,44 @@ describe('EditConnectorFlyout', () => {
       });
     });
 
+    it('updates connector form field with latest value', async () => {
+      const { getByTestId } = appMockRenderer.render(
+        <EditConnectorFlyout
+          actionTypeRegistry={actionTypeRegistry}
+          onClose={onClose}
+          connector={connector}
+          onConnectorUpdated={onConnectorUpdated}
+        />
+      );
+
+      await waitFor(() => {
+        expect(getByTestId('test-connector-text-field')).toBeInTheDocument();
+      });
+
+      await act(async () => {
```

Comment:
I don't believe we need to wrap `userEvent` in `act`: https://github.com/testing-library/user-event/issues/497#issuecomment-729897974